### PR TITLE
net-mgmt/telegraf: add elasticsearch output

### PR DESCRIPTION
**Pull request by mimugmail**
For #780 

I'm not sure about the index_name field, it has to include "%".
Validators? Prohibited? Not important since only for config file generation?

### CODE REVIEW

**Comment by mimugmail** [comment]
Done ..

**Comment by mimugmail** [comment]
Sorry .. not enough coffee or beer .. :/

File: net-mgmt/telegraf/src/opnsense/mvc/app/controllers/OPNsense/Telegraf/forms/output.xml [hunk 1]
```diff
@@ -77,4 +77,28 @@
...
+        <id>output.elastic_timeout</id>
+        <label>Timeout</label>
+        <type>text</type>
+        <help>Write timeout (for the Elasticsearch client), formatted as a string. If not provided, will default to 5s. 0s means no timeout (not recommended).</help>
+    </field>
+    <field>
+        <id>output.elastic_indexname</id>
+        <label>Index Name</label>
+        <type>text</type>
+        <help>Set the index name, default is "telegraf-%Y.%m.%d".</help>
```
Comment (fichtner) [inline net-mgmt/telegraf/src/opnsense/mvc/app/controllers/OPNsense/Telegraf/forms/output.xml]: Still need to remove "default is ..." from here. :)

**Comment by mimugmail** [comment]
Yep :)

**Comment by mimugmail** [comment]
Ok, plz devel pkg for 18.7.1 when possible

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**